### PR TITLE
Change Email to UUID in settings

### PR DESCRIPTION
**Pull request by TOLoneWolf**
- Change Email to Dashed UUID in settings.
- Changed copy Email to copy Dashed UUID.
- Fixed css errors on username and UUID(email) containers.
- Inscreased width of container to stop wrapping.

### CODE REVIEW

File: src/common/modals/Settings/components/General.js [hunk 1]
```diff
@@ -196,6 +196,22 @@ function copy(setCopied, copyText) {
   }, 500);
 }
 
+function dashUuid(UUID) {
+  // UUID is segmented into: 8 - 4 - 4 - 4 - 12
+  // Then dashes are added between.
+
+  // eslint-disable-next-line
```
Comment (blarfoon) [inline src/common/modals/Settings/components/General.js:205]: Why disable eslint?

**Comment by blarfoon** [comment]
Why do we want to show uuid instead of email? I think we might as well show both at this point

File: src/common/modals/Settings/components/General.js (hunk 1 above)
Comment (TOLoneWolf) [reply src/common/modals/Settings/components/General.js:205]: It wants the substrings all over the place instead all nice and lined up.

**Comment by TOLoneWolf** [comment]
I already know my email address and don't need a game launcher displaying it.
UUID is something you might need to know if you need to find your player.dat file on a server.

Anyone who streams and enters to change setting will show off their email address, it might be a private one and not their public one (if they have one) that they want showing.

File: src/common/modals/Settings/components/General.js (hunk 1 above)
Comment (TOLoneWolf) [reply src/common/modals/Settings/components/General.js:205]: This is what prettier/prettier want to do if not disabled.
image

**Pull request merged**